### PR TITLE
Nattinan 01 initial.setup complete!

### PR DESCRIPTION
**Pull request by NaOgit**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    color: #ecf0f1;
+
+}
+
+h1 + p {
+	margin-bottom: 5%;
+    color: #3498db;
+}
+
+.scorebox {float: left;}
```
Comment (richgurney) [inline css/style.css]: Check out formatting here

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+.timebox {float: right;}
+
+.outerbox:after {
+	content: "";
```
Comment (richgurney) [inline css/style.css]: Indentation needs fixing here

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+button:active {
+  transform: translate(0px, 5px);
+  -webkit-transform: translate(0px, 5px);
+  box-shadow: 0px 1px 0px 0px;
+}
+
+.scorebox p, .scorebox span, .timebox p, .timebox span {
+    font-size: 30px;
```
Comment (richgurney) [inline css/style.css]: Make indentation consistent here

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+$(document).ready(function()
+{
```
Comment (richgurney) [inline js/main.js]: This should be on the previous line

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  //Words array
+  var gameWrds = ["Allegation", "Idiosyncritic", "Magnanimous" , "Perfunctory", "Paradox", "Tiresome", "Inadequate", "Jargon", "Appraised", "Reprehensible", "Outrageous", "Atrocious", "Scandalous", "Virtuous", "Dazzling", "Unimpeachable","Unassailed", "Insusceptible", "Vulnerable", "Unparagoned", "Conspicuous", "Perfidiousness", "Aggression", "Hypothesis", "Exasperation", "Reincarnation", "Unpredictable", "Parameter", "Medicore", "Frivolous", "Hippopotamus", "Comprehensive", "Blasphemy", "Psychological", "Sesquipedalian", "Genocide", "Harmony", "Exaggeration", "Jabot", "Circumvallate", "Effulgent", "Ineffable", "Fulguration", "Greensward", "Horripilation", "Lachrymal", "Empyrean", "Crapulent", "Coruscate", "Evanescent", "Fuliginous", "Hymeneal", "Perficious", "Plangent", "Prothalamium", "Suserration", "Temerarious", "Viridescent", "Zephyr", "Hypocrite"];
+  //declaring global variables
+  var displayWord = random();
+  var points = 0;
+  var seconds = 60;
+  var spark = new
+  //countdown function is declared
+  function countdown() {
+           points = 0;
```
Comment (richgurney) [inline js/main.js]: We need consistent indentation through the js file

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    }
+  //Generate random words from array
+  function random() {
+      $(".words").html("");
+      var index = Math.floor(Math.random() * gameWrds.length - 1 );
+    var word = gameWrds[index];
+      for (var i = 0; i < word.length; i++) {
+      return word;
+      }
+    // var myArray = getArray(word);
```
Comment (richgurney) [inline js/main.js]: No commented out code

File: js/main.js [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  //Getting keyboard to give inputs
+  $(document).keydown(function(e){
+      var currentLetter = $("#letter" + currentLetterId);
+      currentLetter.addClass("bg");
+      var keyinput;
+      if(window.event) {
+        // IE
+        keyinput = e.keyCode;
+      } else if(e.which){
+        // Netscape/Firefox/Opera
```
Comment (richgurney) [inline js/main.js]: What does this mean?

File: pages/1.html [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    <script
+    src="https://code.jquery.com/jquery-3.2.1.js"
+    integrity="sha256-DZAnKJ/6XZ9si04Hgrsxu/8s717jcIzLy3oi35EouyE="
+    crossorigin="anonymous"></script>
+    <script type="text/javascript" src="../js/main.js"></script>
+    <link rel="stylesheet" href="../css/style.css">
+  </head>
+  <body>
+  <div class="container">
+		<h1></h1>
```
Comment (richgurney) [inline pages/1.html]: Can you sort out the indentation here please

File: css/style.css (hunk 2 above)
Comment (AEJackson) [reply css/style.css]: fix indentation here

**Pull request merged**